src/microbe_stage/microbe_camera_system.cpp

```diff
@@ -56,8 +56,8 @@ void
 
         if(cameraPos.Members._Position != targetPos) {
 
-            cameraPos.Members._Position =
-                cameraPos.Members._Position.Lerp(targetPos, CAMERA_FLOW);
+            cameraPos.Members._Position = cameraPos.Members._Position.Lerp(
+                targetPos, elapsed / CAMERA_FLOW);
```

Comment:
Does it make sense to have it this way? Why not increase the CAMERA_FLOW constant and do it like `CAMERA_FLOW * elapsed` that way my brain could understand what this is doing much easier?

Reply:
Yes it's more sensible that way, but the CAMERA_FLOW value has to be like 7 or 8.

Reply:
I don't see a problem with setting CAMERA_FLOW to be 7. There could also be another constant multiplied together with them to keep the CAMERA_FLOW small.

Reply:
Ok, I need to test the values for a sec